src/components/FilterCard.vue

```diff
@@ -86,14 +86,14 @@ export default {
     }
   },
   computed: {
-    wildcard() {
+    isWildcardOption() {
```

Comment:
I notice that changing this didn't require any tests to be updated with the new name.  Does this method need testing?

Reply:
There isnt a specific test for it but it is indirectly tested here: https://github.com/sanger/traction-ui/pull/1161/files#diff-b5412883d88e30421c0da822ffc118c6e78d92461ad80bc91b19a7510785b0a6R68. I can add a test if you think one would be useful.

Reply:
That's fair.  I guess the implementation details aren't important to test.  Only that the user selects an option to be a wildcard filter and that is the one that gets marked as such.  I would recommend updating that test though to make each of the three filters the wildcard option.  You can use `it.each` (https://vitest.dev/api/#test-each) to run the test multiple times with the value of the filter to enable.  i.e. `[1, 2, 3]`.  Just testing the first one means the coverage isn't optimal.  Glad that there is a test though!

Reply:
yeah that sounds good, I have added something along those lines.